### PR TITLE
test(manager): added manager tests that use enterprise ami

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by ilya-rarov — APPROVED** [review]
LGTM

File: jenkins-pipelines/manager/enterprise-ami-manager-sanity.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    test_name: 'mgmt_cli_test.MgmtCliTest.test_manager_sanity',
+    test_config: 'test-cases/manager/manager-regression-multiDC-set-distro.yaml',
+
+    timeout: [time: 240, unit: 'MINUTES'],
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+
+    downstream_jobs_to_run: 'enterprise-ami-upgrade-test'
```
Comment (rayakurl) [inline jenkins-pipelines/manager/enterprise-ami-manager-sanity.jenkinsfile:21]: shouldn't it be the same name as the jenkinsfile? (enterprise-ami-manager-upgrade.jenkinsfile)?

Comment (fruch) [reply jenkins-pipelines/manager/enterprise-ami-manager-sanity.jenkinsfile:21]: I think this is a reminder that manager jobs are created by hand, and not via a hydra command (like in scylla releases). 
so the names might vary based on whom created the jobs.

also the job name is the jenkinsfile basename + `-test`

the automated code path would have created a job named: `enterprise-ami-manager-upgrade-test`

File: jenkins-pipelines/manager/enterprise-ami-manager-upgrade.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+managerPipeline(
+    manager: true,
+    backend: 'aws',
+    region: 'us-east-1',
```
[the remark this answers is not in the document]
Comment (ShlomiBalalis) [reply jenkins-pipelines/manager/enterprise-ami-manager-upgrade.jenkinsfile:9]: It always has been. We wanted to test both single dc and multi dc, so we made the sanity multi dc and the rest of the tests single dc.

File: jenkins-pipelines/manager/enterprise-ami-manager-sanity.jenkinsfile (hunk 1 above)
Comment (ShlomiBalalis) [reply jenkins-pipelines/manager/enterprise-ami-manager-sanity.jenkinsfile:21]: That is true

**Review by rayakurl — APPROVED** [review]

**Comment by rayakurl** [comment]
@fruch - can you please merge? thanks

**Pull request merged**